### ISSUE TITLE
Update courses with HoloToolkit to match the actual toolkit structure better

### ISSUE THREAD

**Issue by angelaHillier** [Microsoft/HolographicAcademy#3]
We had a user on the forums mention that HoloToolkit used in academy courses does not match the HoloToolkit available on GitHub. This mis-match is causing confusion when developers attempt to use the HoloToolkit in their projects.
https://forums.hololens.com/discussion/389/using-the-holotoolkit-cursors

We should update our academy courses to use the latest HoloToolkit (and folder structure)

**Comment by keveleigh** [comment]
I'll jump on this, unless somebody else has already made progress.

**Comment by angelaHillier** [comment]
Thanks Kurtis! I'll do Spatial Mapping, since I'm making other changes in that course :)

**Comment by erichahn** [comment]
I'm glad that you're keeping the tutorials up to date. It'd be nice to have the tutorial explicitly reference this git repo and inform users to make sure they're up to date while trying to create their own apps from the tutorial concepts.

I ran into an issue today where the old tutorial referenced GazeEntered() and the new toolkit uses OnGazeEnter(). Needless to say I felt incredibly dumb after a couple hours wasted.

**Comment by NeerajW** [comment]
@erichahn totally understand the confusion and appreciate the feedback.
However, as the HoloToolkit grows, it will be very hard to keep learning classes always in sync with the open source projects. Hence that will be a non-goal for these tutorials.

These are for learning concepts and it is very hard to guarantee that some changes in the public facing project will not break these.

If you face any issues please do feel free to reach out to us on forums.hololens.com. We all would be more than happy to get you unblocked.

Hope this helps clear things up a little.

**Comment by timGerken** [comment]
Maybe we can eliminate some of the confusion by not calling the folder "HoloToolkit" in the tutorials.
Some options:
- Toolkit
- SimpleToolkit
- Tools

**Comment by angelaHillier** [comment]
Yeah, Neeraj and I had discussed renaming this in the past. I like 'Toolkit' or 'CustomHoloToolkit' or 'HoloToolkit_Custom'. The problem is that we now have a lot of recorded videos that tell you to go to the 'HoloToolkit' folder. If we keep the name *close* enough, then it shouldn't cause too much confusion.

I've tried to keep the SpatialMapping course up-to-date with the HoloToolkit, as it does actually teach people how to use components from the kit.

It would be good to update the Sharing course with a newer version of the HoloToolkit too, as I know there have been a lot of fixes that have gone into the toolkit since we last synced that project. The issue with this one is that the HoloToolkit used in the course has a completely different folder structure, so it doesn't map well.

**Comment by NeerajW** [comment]
Agreed with Angela on keeping the name similar to HoloToolkit so the instructions and intent is not lost.
Perhaps we can call it:

1. HoloToolkit_Custom
2. HoloToolkit_Academy

**Comment by timGerken** [comment]
How about HoloToolkit_Academy_### where ### is the course number?
See [this thread](https://forums.hololens.com/discussion/2726/academy-211-handsmanager-not-working-gesturemanager-has-no-definition-for-instance#latest) for real-world user confusion :)

**Comment by NeerajW** [comment]
Sounds great.

**Comment by angelaHillier** [comment]
We decided to 'customize' the HoloToolkit for each course. This is being addressed in issue #16.
